### DISCUSSION TITLE
[Beginner] - Should I use symfony/routing if I only have access to GET params?

### DISCUSSION THREAD

**Discussion by chris-ant** [symfony/symfony#52691] [Q&A] [answered]
Hello there, 

I'm going through https://symfony.com/doc/current/create_framework/index.html because I want to integrate Symphony request/response into a legacy app, but I can only do so through this app's own front.php kind of file. 

Since I don't control the front access and load my app via $_GET params, I can't have a path after $_GET params. I can have other $_GET params just fine. 

So my question is, do I need the symfony/routing? Or just use the basic example from the tutorial, or a simple custom class. 

Is the routing component responsible just for loading the Controller? Or I'm missing out on other features as well that will come apparent later on. 

For context, this app will have forms in it that will save data to the database. 

Thank you, and I hope the question isn't too silly.
Best Regards,
Chris

**Comment by ghost** [comment, marked as answer]
If you only have access to $_GET global variable then there isn't much you can do. To use Symfony req/res flow you have to create a Request object. You can't use the most common `createFromGlobals()` method but you can use `create()` however you have to have URI because it's a required parameter.

> For context, this app will have forms in it that will save data to the database.

The question is: do you need a framework or do you need only form component.

**Comment by chris-ant** [comment]
Ah, I hadn't considered using just the form component! I only have about three screens at most that I need, so I can simply put those together without worrying about the request/response flow.

I'll explore this option. Thank you!
